docs/sysext.md

```diff
@@ -92,4 +92,6 @@ BaseTrees=%O/base
 ```
 
 If we now run `mkosi -f` again, we'll find `initrd.cpio.zst` in
-`mkosi.output` with its accompanying extension still in `btrfs.raw`.
+`mkosi.output` with its accompanying extension still in `btrfs.raw`. If
```

Comment:
Nit: the line's pretty short anyway, so you can reformat the whole paragraph, otherwise
```suggestion
`mkosi.output` with its accompanying extension still in `btrfs.raw`.
If
```
would make the diff shorter by the unchanged line.